src/installer/installer.go

```diff
@@ -722,7 +722,11 @@ func (i *installer) cleanupDevice(device string) error {
 		}
 	}
 
-	return i.ops.RemoveAllPVsOnDevice(device)
+	if err = i.ops.RemoveAllPVsOnDevice(device); err != nil {
+		return err
```

Comment:
Do we really want a hard error here? Let's keep it best-effort

Reply:
Or let's just remove [this](https://github.com/openshift/assisted-installer/blob/8027b73de9c36396f90530eb2b5481dc4b3532db/src/installer/installer.go#L99) line

Reply:
hmm yes